pr2eus_tutorials/pr2eus_tutorials.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git:
```

Comment:
wnen you create `.rosinstall` file, please add why you need that package from source, with PR URLs. see -> https://github.com/jsk-ros-pkg/jsk_demos/blob/master/detect_cans_in_fridge_201202/detect_cans_in_fridge_201202.rosinstall